packages/docusaurus-theme-classic/src/theme/BlogPostPage/index.tsx

```diff
@@ -10,18 +10,22 @@ import React from 'react';
 import Layout from '@theme/Layout';
 import BlogPostItem from '@theme/BlogPostItem';
 import BlogPostPaginator from '@theme/BlogPostPaginator';
+import {DocTOC} from '@theme/DocItem';
```

Comment:
This feels wrong as we're making the blog features rely on the docs feature. Maybe we should create a generic TOC component and have `@theme/DocItem` and `@theme/BlogPostPage` import it instead

Reply:
Makes sense. I've made a TOC component and moved things there